dokka-subprojects/plugin-base/src/main/resources/dokka/templates/base.ftl

```diff
@@ -32,7 +32,23 @@
         <@header.display/>
         <div id="container">
             <div class="sidebar" id="leftColumn">
-                <div class="sidebar--inner" id="sideMenu"></div>
+                <div class="dropdown theme-dark_mobile" data-role="dropdown" id="toc-dropdown">
```

Comment:
Yes, I've just double checked, custom templates will still work.
However it will be needed update custom templates to align them with the changes in our templates, we'll write about it in the release notes and provide links to the templates in this PR as a reference

Reply:
Thanks!
And what will be the result if the templates will not be updated?
I mean, for now the only thing worries me here is `kotlinx.*` libraries because their templates are stored in [kotlin-web-site](https://github.com/JetBrains/kotlin-web-site) repository and used by all other kotlinx libraries.

Let's imagine a case that `kotlinx.coroutines` will update to Dokka 2.0.0 but `kotlinx.serialization` will still use Dokka 1.9.20 - they will both use the same templates - how the final result will look for both libraries?

Reply:
We will need to update api references for kotlinx- libraries simultaniously